src/modules/physics/box2d/Physics.h

```diff
@@ -359,6 +359,10 @@ class Physics : public Module
 	 **/
 	static b2AABB scaleUp(const b2AABB& aabb);
 
+	/**
```

Comment:
This should be filled out (with two `*`'s in the final line to match the other comments).

I don't think each parameter needs to be listed, but a short description would be nice at least